nest/02.nest.md

```diff
@@ -72,6 +77,20 @@ So, if this package was a text editor for example, the files you created or edit
 
 However, this may not remove all the package's dependencies, because some of them might still be used by another installed package.
 
+```
```

Comment:
Missing an `Example:` or stg similar

Reply:
There is `Example of use:` on the line just above this one

Reply:
To me, the previous line is blank. Am I blind or are we not talking of the same thing?

Reply:
To me, it's not D:
anyway, my next commit should fix this